### PR TITLE
Add User Story

### PR DESCRIPTION
**Pull request by briankariuki**
This pull request add a `story_live` page which can be used to create stories for a user. @wintermeyer please do not close this, editing of the story feature is still work in progress. Changes are very much welcome.

image

### CODE REVIEW

**Review by wintermeyer — APPROVED** [review]

**Comment by wintermeyer** [comment]
@briankariuki What do you mean by "please do not close this"? Should I merge this PR into main or not yet?

**Comment by wintermeyer** [comment]
This should be a pull down. The user can not create his/her own headline. He/She can only select one of the available headlines.

Bildschirmfoto 2024-03-16 um 16 24 45

File: lib/animina_web/live/story_live.ex [hunk 1]
```diff
@@ -33,15 +42,10 @@ defmodule AniminaWeb.StoryLive.Create do
...
-          ],
-          photo: [
-            resource: Photo,
-            create_action: :create
           ]
         ]
       )
-      |> AshPhoenix.Form.add_form([:headline])
-      |> AshPhoenix.Form.add_form([:photo])
+      |> AshPhoenix.Form.add_form([:headline], params: %{subject: "About me"}, validate?: false)
```
Comment (wintermeyer) [inline lib/animina_web/live/story_live.ex]: This and other similar texts need to be gettexted.

**Comment by briankariuki** [comment]
> This should be a pull down. The user can not create his/her own headline. He/She can only select one of the available headlines.
> 
> Bildschirmfoto 2024-03-16 um 16 24 45

@wintermeyer please review the latest commit

**Pull request merged**